kanidmd/src/lib/core/https/oauth2.rs

```diff
@@ -241,6 +241,11 @@ async fn oauth2_authorise(
             Ok(tide::Response::new(tide::StatusCode::Unauthorized))
         }
         Err(e) => {
+            debug!(
```

Comment:
Should be error not debug if this happens IMO.

Reply:
fixed